### DISCUSSION TITLE
Do we actually need to "Install Rooted Firmware" these days?

### DISCUSSION THREAD

**Discussion by steveharman** [Guilouz/Creality-Helper-Script-Wiki#461] [General]
In the Wiki there's a page that explains how to install "rooted firmware" via an SD card:  https://guilouz.github.io/Creality-K1-Series/firmwares/install-and-update-rooted-firmware/

Assuming a printer _is already on_ the latest firmware (v1.3.3.5) is there any benefit to flashing rooted firmware via the SD card? Or can we just use the "Enable root" option via the printer's LCD?

**Comment by halTWO3** [comment]
Using sd card is just faster and easier, also for those who doesn't have access to the internet. The latest firmware which is available at creality's site is already pre rooted. You can just enable root access via the menu.

**Reply by steveharman** [reply]
Understood. But the question was really related to the case of "_Assuming a printer is already on the latest firmware (v1.3.3.5_)".

I just wanted to understand if there was a requirement or need to actually flash the "rooted firmware" that was linked to, when I was already setup with v1.3.3.5 and had root enabled.

**Reply by halTWO3** [reply]
The firmware that was link to is the official firmware. Same firmware the printer auto updates to via internet. So no, no benefit. They are basically the same.

**Reply by steveharman** [reply]
Got it.    I wonder if it might be worth having a small note about this on the wiki page?  atm it reads like there is some kind of special rooted firmware that needs applying, which would be a bit of a waste of time for anyone on v1.3.3.5 who just needs to check a box via the LCD.  ;-)

**Reply by halTWO3** [reply]
Hehe /agreed
